deepmd/pt/model/descriptor/se_a.py

```diff
@@ -225,11 +249,58 @@ def reinit_exclude(
         """Update the type exclusions."""
         self.sea.reinit_exclude(exclude_types)
 
+    def enable_compression(
+        self,
+        min_nbor_dist: float,
+        table_extrapolate: float = 5,
+        table_stride_1: float = 0.01,
+        table_stride_2: float = 0.1,
+        check_frequency: int = -1,
+    ) -> None:
+        """Receive the statisitcs (distance, max_nbor_size and env_mat_range) of the training data.
+
+        Parameters
+        ----------
+        min_nbor_dist
+            The nearest distance between atoms
+        table_extrapolate
+            The scale of model extrapolation
+        table_stride_1
+            The uniform stride of the first table
+        table_stride_2
+            The uniform stride of the second table
+        check_frequency
+            The overflow check frequency
+        """
```

Comment:
Please add this method to `BaseBaseModel`.

https://github.com/deepmodeling/deepmd-kit/blob/39cddd4832873ca89c3b22e01e6edc1101f1de54/deepmd/dpmodel/model/base_model.py#L23-L24

We can make this method as a standard API.

Reply:
> And I have a question that why not add enable_compression to deepmd/dpmodle/descriptor/make_base_descriptor.py

... That's what I mean. But anyway, Model and Atomic Model also need a similar API.

Reply:
You declare it as `@abstractmethod`, which means it needs to exist every class. Alternatively, raise NotImplementError in the base class